### PR TITLE
fix(samples): v1 and v1beta1 region tags collide

### PR DESCRIPTION
**Pull request by bcoe**
## Description

The `v1` healthcare region tags collide with the `v1beta1` region tags.

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by busunkim96 — APPROVED** [review]

**Comment by kurtisvg** [comment]
I think at some point this was intended, and the plan was to remove the v1beta4 samples once v1 was complete. 

@noerog PTAL

**Comment by noerog** [comment]
I deleted the majority of the v1beta1 code, but there are still some methods only available in v1beta1 that I want to keep, those being:

healthcare_conditional_update_resource
healthcare_conditional_patch_resource
healthcare_conditional_delete_resource

We could actually just delete all of the other region tags in v1beta1/fhir/ since none of them are being used except for the 3 above.

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 _BASE_URL = "https://healthcare.googleapis.com/v1beta1"
 
 
-# [START healthcare_get_session]
+# [START healthcare_v1beta1_get_session]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:26]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 2]
```diff
@@ -46,10 +46,10 @@ def get_session(service_account_json):
     return session
 
 
-# [END healthcare_get_session]
+# [END healthcare_v1beta1_get_session]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:49]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 3]
```diff
@@ -46,10 +46,10 @@ def get_session(service_account_json):
...
 
 
-# [START healthcare_create_resource]
+# [START healthcare_v1beta1_create_resource]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:52]:
```suggestion

```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 4]
```diff
@@ -82,10 +82,10 @@ def create_patient(
     return response
 
 
-# [END healthcare_create_resource]
+# [END healthcare_v1beta1_create_resource]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:85]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 5]
```diff
@@ -82,10 +82,10 @@ def create_patient(
...
 
 
-# [START healthcare_create_encounter]
+# [START healthcare_v1beta1_create_encounter]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:88]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 6]
```diff
@@ -134,10 +134,10 @@ def create_encounter(
     return response
 
 
-# [END healthcare_create_encounter]
+# [END healthcare_v1beta1_create_encounter]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:137]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 7]
```diff
@@ -134,10 +134,10 @@ def create_encounter(
...
 
 
-# [START healthcare_create_observation]
+# [START healthcare_v1beta1_create_observation]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:140]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 8]
```diff
@@ -183,10 +183,10 @@ def create_observation(
     return response
 
 
-# [END healthcare_create_observation]
+# [END healthcare_v1beta1_create_observation]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:186]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 9]
```diff
@@ -183,10 +183,10 @@ def create_observation(
...
 
 
-# [START healthcare_delete_resource]
+# [START healthcare_v1beta1_delete_resource]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:189]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_resources.py [hunk 10]
```diff
@@ -218,7 +218,7 @@ def delete_resource(
     return response
 
 
-# [END healthcare_delete_resource]
+# [END healthcare_v1beta1_delete_resource]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_resources.py:221]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 11]
```diff
@@ -20,7 +20,7 @@
 from googleapiclient.errors import HttpError
 
 
-# [START healthcare_get_client]
+# [START healthcare_v1beta1_get_client]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:23]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 12]
```diff
@@ -44,10 +44,10 @@ def get_client(service_account_json):
     )
 
 
-# [END healthcare_get_client]
+# [END healthcare_v1beta1_get_client]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:47]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 13]
```diff
@@ -44,10 +44,10 @@ def get_client(service_account_json):
...
 
 
-# [START healthcare_create_dataset]
+# [START healthcare_v1beta1_create_dataset]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:50]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 14]
```diff
@@ -71,10 +71,10 @@ def create_dataset(service_account_json, project_id, cloud_region, dataset_id):
         return ""
 
 
-# [END healthcare_create_dataset]
+# [END healthcare_v1beta1_create_dataset]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:74]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 15]
```diff
@@ -71,10 +71,10 @@ def create_dataset(service_account_json, project_id, cloud_region, dataset_id):
...
 
 
-# [START healthcare_delete_dataset]
+# [START healthcare_v1beta1_delete_dataset]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:77]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 16]
```diff
@@ -93,10 +93,10 @@ def delete_dataset(service_account_json, project_id, cloud_region, dataset_id):
         return ""
 
 
-# [END healthcare_delete_dataset]
+# [END healthcare_v1beta1_delete_dataset]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:96]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 17]
```diff
@@ -93,10 +93,10 @@ def delete_dataset(service_account_json, project_id, cloud_region, dataset_id):
...
 
 
-# [START healthcare_create_fhir_store]
+# [START healthcare_v1beta1_create_fhir_store]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:99]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 18]
```diff
@@ -121,10 +121,10 @@ def create_fhir_store(
     return response
 
 
-# [END healthcare_create_fhir_store]
+# [END healthcare_v1beta1_create_fhir_store]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:124]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 19]
```diff
@@ -121,10 +121,10 @@ def create_fhir_store(
...
 
 
-# [START healthcare_delete_fhir_store]
+# [START healthcare_v1beta1_delete_fhir_store]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:127]:
```suggestion
```

File: healthcare/api-client/v1beta1/fhir/fhir_stores.py [hunk 20]
```diff
@@ -148,7 +148,7 @@ def delete_fhir_store(
     return response
 
 
-# [END healthcare_delete_fhir_store]
+# [END healthcare_v1beta1_delete_fhir_store]
```
Comment (leahecole) [inline healthcare/api-client/v1beta1/fhir/fhir_stores.py:151]:
```suggestion
```

**Review by leahecole — COMMENTED** [review]
> I deleted the majority of the v1beta1 code, but there are still some methods only available in v1beta1 that I want to keep, those being:
> 
> healthcare_conditional_update_resource
> healthcare_conditional_patch_resource
> healthcare_conditional_delete_resource
> 
> We could actually just delete all of the other region tags in v1beta1/fhir/ since none of them are being used except for the 3 above.

Added suggested changes for removal @bcoe based on this comment from @noerog

**Review by noerog — APPROVED** [review]

**Comment by bcoe** [comment]
@parthea I believe should be ready to land.

**Pull request merged**